CMakeLists.txt

```diff
@@ -224,8 +226,8 @@ if ( KWIVER_ENABLE_MATLAB )
   file( APPEND "${KWIVER_SETUP_SCRIPT_FILE}" "export LD_LIBRARY_PATH=${Matlab_LIBRARY_DIR}:$LD_LIBRARY_PATH\n" )
 endif()
 
-if ( fletch_ROOT )
-  file( APPEND "${KWIVER_SETUP_SCRIPT_FILE}" "export LD_LIBRARY_PATH=${fletch_ROOT}/lib:$LD_LIBRARY_PATH\n" )
+if ( fletch_FOUND )
+  file( APPEND "${KWIVER_SETUP_SCRIPT_FILE}" "export LD_LIBRARY_PATH=${fletch_DIR}/lib:$LD_LIBRARY_PATH\n" )
```

Comment:
The fletchConfig.cmake file exports fletch_ROOT to point to the install directory. The fletch_DIR points to the directory that contains fletchConfig.cmake and is used by find_module(). These two variables may point to the same directory, but they do not have to. I think adding fletch_ROOT to the paths is more robust.